### PR TITLE
:wrench: DOP-3988

### PR DESCRIPTION
**Pull request by caesarbell**

### CODE REVIEW

**Review by seungpark — APPROVED** [review]
this LGTM 👍 tested with DOP-3988 branch of parser

File: source/index.txt [hunk 1]
```diff
@@ -79,85 +82,79 @@ MongoDB Documentation
...
       :icon-alt: Go Driver icon
 
    .. card::
       :headline: Java
       :url: https://www.mongodb.com/docs/drivers/java-drivers/
-      :icon: /icons/java.svg
+      :icon: /images/icons/java.svg
       :icon-alt: Java Driver icon
    
-   .. card::
```
Comment (seungpark) [inline source/index.txt:109]: minor note! why did we remove Kotlin?

Comment (caesarbell) [reply source/index.txt:109]: @seungpark good question. It wasn't in the [Figma](https://www.figma.com/file/SxKPGkhEJ8ixM9giIPEuJC/Docs-Homepage-Redesign?node-id=1351%3A59951&mode=dev) designs.

Comment (seungpark) [reply source/index.txt:109]: Odd! would confirm with design before removing! Seems like its there on desktop view but missing in tablet/mobile. Suspecting this was accidental!

**Review by schmalliso — APPROVED** [review]

File: source/index.txt (hunk 1 above)
Comment (schmalliso) [reply source/index.txt:109]: Lets put Kotlin back, but otherwise, LGTM.

**Pull request merged**